triage/os_version_linux.go

```diff
@@ -0,0 +1,27 @@
+package triage
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func printOSVersion() {
+	file, err := os.Open("/etc/os-release")
+	if err != nil {
+		fmt.Println("Error reading OS release:", err)
+		return
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasPrefix(line, "PRETTY_NAME=") {
+			fmt.Println("OS:", strings.Trim(line[12:], `"`))
```

Comment:
you should use https://pkg.go.dev/strings#Split on '=' instead of a fixed offset, but it's ok to merge as is if you don't want to change it.